android/build.gradle

```diff
@@ -5,14 +5,49 @@ allprojects {
     }
 }
 
-rootProject.buildDir = '../build'
+allprojects {
+    tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
+        kotlinOptions {
+            jvmTarget = "1.8"
+        }
+    }
+}
+
+rootProject.buildDir = "../build"
+subprojects {
```

Comment:
Some libraries still haven't added the `namespace` so we need to fix it manually, can be removed once they will be updated